practice-one/src/components/footer.html

```diff
@@ -0,0 +1,44 @@
+<footer class="footer-container">
+  <div class="website-summary-container">
+    <div class="brief-description">
+      <h3 class="brief-heading">Foodhut</h3>
+      <p class="brief-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor </p>
+      <div class="brief-icon-container">
+        <div class="social-media-icon"><img src="./assets/images/facebook.svg" alt="Facebook icon"></div>
+        <div class="social-media-icon"><img src="./assets/images/instagram.svg" alt="Instagram icon"></div>
+        <div class="social-media-icon"><img src="./assets/images/twitter.svg" alt="Twitter icon"></div>
+      </div>
+    </div>
+
+    <div class="company-information-container">
+      <h3 class="company-information-heading">About Us</h3>
+      <ul class="company-information-list">
+        <li class="company-information-item"><a href="">About Us</a></li>
+        <li class="company-information-item"><a href="">Service Us</a></li>
+        <li class="company-information-item"><a href="">Contact</a></li>
+        <li class="company-information-item"><a href="">Company</a></li>
```

Comment:
Missing href, any idea for this? You can provide a mock value for this and comment above for clear